diff --git a/node_modules/@stencil/core/compiler/stencil.js b/node_modules/@stencil/core/compiler/stencil.js
index 13230ac..c14b23e 100644
--- a/node_modules/@stencil/core/compiler/stencil.js
+++ b/node_modules/@stencil/core/compiler/stencil.js
@@ -267325,7 +267325,8 @@ var bundleOutput = async (config, compilerCtx, buildCtx, bundleOpts) => {
   try {
     const rollupOptions = getRollupOptions(config, compilerCtx, buildCtx, bundleOpts);
     const rollupBuild = await rollup(rollupOptions);
-    compilerCtx.rollupCache.set(bundleOpts.id, rollupBuild.cache);
+    // PATCHED: don't store Rollup cache (stenciljs/core#6253)
+    // compilerCtx.rollupCache.set(bundleOpts.id, rollupBuild.cache);
     return rollupBuild;
   } catch (e) {
     if (!buildCtx.hasError) {
@@ -267443,7 +267444,7 @@ var getRollupOptions = (config, compilerCtx, buildCtx, bundleOpts) => {
     treeshake: getTreeshakeOption(config, bundleOpts),
     preserveEntrySignatures: (_e = bundleOpts.preserveEntrySignatures) != null ? _e : "strict",
     onwarn: createOnWarnFn(buildCtx.diagnostics),
-    cache: compilerCtx.rollupCache.get(bundleOpts.id),
+    cache: false, // PATCHED: disable Rollup cache to fix watch rebuild hang (stenciljs/core#6253)
     external: config.rollupConfig.inputOptions.external,
     maxParallelFileOps: config.rollupConfig.inputOptions.maxParallelFileOps
   };
